languagetool-gui-commons/src/main/java/org/languagetool/gui/ConfigurationDialog.java

```diff
@@ -300,7 +287,7 @@ public void actionPerformed(@SuppressWarnings("unused") ActionEvent actionEvent)
     jPane.add(getTreeButtonPanel(0), cons);
 
     tabpane.addTab(messages.getString("guiGrammarRules"), jPane);
-    
```

Comment:
Could you configure your IDE so that it doesn't modify existing whitespace? This makes diffs difficult to read,